### PR TITLE
Add sdg seed data 

### LINKED ISSUE #250: add seed data for sdg
### Overview

seed data for #70

### Action Items

- [ ] Find the table and create the json file
- [ ] convert the json file to python script
- [ ] Combine the script in migration

### Resources/Instructions

https://hackforla.github.io/peopledepot/how-to/create-initial-data-migrations/

### PR DESCRIPTION
**Pull request by freaky4wrld**
Fixes #250 

### What changes did you make?

- Added json file for seed data
- Added the converted python file
- Added the migrations file to add seed data

### Why did you make the changes (we will use this info to test)?

- To add the seed data for sdg model

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

- No visual changes

### CODE REVIEW

**Comment by freaky4wrld** [comment]
@fyliu I'm having trouble with the seed data as it is exceeding the flake8 199 character criteria, can you please provide the instruction

**Comment by fyliu** [comment]
@freaky4wrld you can add `data/migrations` to the ignore list for flake8 in the pre-commit config. See `.pre-commit-config.yaml` in the project root for an example of multiple excludes and also the hook to modify.

File: app/data/migrations/0003_sdg_seed.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    for uuid, name, description, image in items:
+        Sdg.objects.create(uuid=uuid, name=name, description=description, image=image)
+
+
+class Migration(migrations.Migration):
+    initial = True
+    dependencies = [("data", "0002_practicearea_seed")]
+
+
+operations = [migrations.RunPython(run, migrations.RunPython.noop)]
```
Comment (fyliu) [inline app/data/migrations/0003_sdg_seed.py:115]: This line should be part of `Migration`

**Review by fyliu — CHANGES_REQUESTED** [review]
Looks good. Just a small thing I saw that needs fixing.

I saw that the coverage report says there's many lines in `0003_sdg_seed.py` not covered. But it didn't complain about `0002_practicearea_seed.py`. So I compared the files and found that the last line in `0003_...` isn't in the function to be called during a migration.

**Review by fyliu — COMMENTED** [review]
Works well.

Maybe delte the `sdg_seed.py` since it's incorporated into the migration file.

**Review by fyliu — APPROVED** [review]
Looks good

**Pull request merged**